project.clj

```diff
@@ -1,7 +1,7 @@
 (defn js-dir
-      "Prefix with full JavaScript directory."
-      [path]
-      (str "resources/public/js/" path))
+  "Prefix with full JavaScript directory."
+  [path]
+  (str "resources/public/js/" path))
```

Comment:
Maybe move this to a common urls file?

Reply:
I'm not certain if other namespaces can be referred to the `project.clj` file; since it does not use an `ns` macro but I'll look into it